conformance/utils/suite/features.go

```diff
@@ -61,7 +61,7 @@ const (
 var GatewayExtendedFeatures = sets.New(
```

Comment:
Mind adding a comment above this and the corresponding HTTPRoute list to clarify that this is _only_ Extended features and intentionally does not include Core features. That probably seems obvious, but it's likely worth clarifying because it hasn't always been that way.